src/screens/Customers/AddMeterReading.tsx

```diff
@@ -87,7 +87,7 @@ function AddMeterReading(props: AddMeterReadingProps) {
   }
 
   const cardInfo = [{
-    number: startingMeterAmount.toString(),
+    number: round(startingMeterAmount),
```

Comment:
Nit: I guess it was always like this but it feels rather unintuitive that the `number` prop is actually a `string` type. Not totally necessary but I feel like it would be more intuitive to rename to something like `value`.